PATCH-P0-FIX-UPSTREAM: Fix building in different directory
BUGS: upstream#45616

Makes it possible to run compilation somewhere else then source code is.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: scripts/Makefile.am
===================================================================
--- scripts/Makefile.am.orig
+++ scripts/Makefile.am
@@ -120,7 +120,7 @@ mysql_fix_privilege_tables.sql: mysql_sy
 	@echo "-- This file is automaticly generated from" > $@ 
 	@echo "-- mysql_system_tables.sql & mysql_system_tables_fix.sql" >> $@
 	@echo "" >> $@
-	@cat mysql_system_tables.sql mysql_system_tables_fix.sql >> $@
+	@cat $(top_srcdir)/scripts/mysql_system_tables.sql $(top_srcdir)/scripts/mysql_system_tables_fix.sql > $@
 
 #
 # Build mysql_fix_privilege_tables_sql.c from
@@ -133,7 +133,7 @@ mysql_fix_privilege_tables_sql.c: comp_s
 	sleep 2
 	$(top_builddir)/scripts/comp_sql$(EXEEXT) \
 	  mysql_fix_privilege_tables \
-	    $(top_srcdir)/scripts/mysql_fix_privilege_tables.sql $@
+	    $(top_builddir)/scripts/mysql_fix_privilege_tables.sql $@
 
 
 SUFFIXES = .sh
